spec/factories/orders.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+STATES = %i[new failed completed].freeze
```

Comment:
Czy nie lepiej przenieść to do modelu?

Reply:
oczywiscie ze lepiej! factorka nie jest miejscem do trzymania takich informacji